.github/workflows/build-python.yml

```diff
@@ -18,9 +18,11 @@ jobs:
       CTEST_PARALLEL_LEVEL: 2
       CMAKE_BUILD_TYPE: ${{ matrix.build_type }}
       PYTHON_VERSION: ${{ matrix.python_version }}
+      BOOST_VERSION: 1.72.0
+      BOOST_EXE: boost_1_72_0-msvc-14.2      
 
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
What was the need for this? We would like the CI to fail immediately on errors, whereas full error reporting should ideally be done locally.

Reply:
> What was the need for this? We would like the CI to fail immediately on errors, whereas full error reporting should ideally be done locally.

On this one, we run multiples os system, with different configuration, that one configure is not connected to another.
On my develop on windows, I did stop, to see if the other things is ok, even the windows was fail. when the flag is true, it really hard for dev.
I already give you that if you push another commit to the PR, all the previous job will terminate automatic. 
This is the ideal for develop.

Reply:
The CI will fail when only one job is failing. And you cannot see the other job results.
You assuming that people dev on local, then publish the PR. but the truth is people develops and relay on CI for dev like me.

Reply:
Sorry but we have limited CI resources since we don't get paid for this project. You should not be using our CI for your testing purposes and bottlenecking other PRs.

Reply:
I am not using your CI, I am using my CI for dev.

Reply:
Then please undo this change before pushing here. You can make the change for your branch, test on your CI and then undo the change before pushing it here. :)